lms/djangoapps/mobile_api/settings.py

```diff
@@ -0,0 +1,7 @@
+'''
+    Settings for Mobile Api
+'''
+
+FB_SETTINGS = {'_FACEBOOK_APP_SECRET': "6c26348ef355f53a531890e980ddc731",
+    '_FACEBOOK_APP_ID': "735343629893573",
+    '_FACEBOOK_API_VERSION': "v2.2"}
```

Comment:
Thanks - I didn't know what you were excepting until now. 
Note that in common.py I left the original app keys for testing purposes but they will be obsolete once you merge the PR since this isn't the real app anyway.